### PR TITLE
feat : added home page

### LINKED ISSUE #1: Home Page
- [ ] Title Header
- [ ] Categories 
- [ ] Trending

### PR DESCRIPTION
**Pull request by thenorthkun**
Added home page layout along with nav bar

**Preview :** https://fezmart-git-home-page-thenorthkun.vercel.app/

### CODE REVIEW

File: index.css [hunk 1]
```diff
@@ -0,0 +1,188 @@
...
+}
+
+.hero-section {
+  margin-top: 4.2rem;
+  padding-left: 4rem;
+  padding-right: 4rem;
+  background-size: cover;
+}
+
+.h2 {
```
Comment (j24m) [inline index.css:16]: you can maybe give a class to this element?

**Review by j24m — APPROVED** [review]
Good work. UI looks neat and so does the code.

File: index.css [hunk 2]
```diff
@@ -0,0 +1,189 @@
...
+.card-vertical .card-title{
+    font-size: 1.5rem;
+    letter-spacing: -0.04em;
+    font-weight: 600;
+}
+
+.card-vertical .card-subtitle{
+    font-size: 0.9rem;
+    font-weight: 350;
+    padding-top: 4px;
```
Comment (amantulsyan35) [inline index.css:148]: I think using both rem and px makes it inconsistent, can keep it uniform

File: index.css [hunk 3]
```diff
@@ -0,0 +1,189 @@
...
+    padding-left: 5px;
+    color: var(--red);
+}
+
+.btn-box{
+    padding: 0.875rem;
+    margin-top: -1rem;
+    display: flex;
+    justify-content: center;
+    align-items: center;
```
Comment (amantulsyan35) [inline index.css:177]: can make a seperate flex utility class for this

**Pull request merged**